packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ThumbnailCreator.java

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.imagepicker;
+
+import android.graphics.Bitmap;
+import android.media.ThumbnailUtils;
+import android.os.AsyncTask;
+import android.provider.MediaStore;
+import androidx.core.util.Consumer;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class ThumbnailCreator {
```

Comment:
public seems to be an overkill for this class if it is only used inside the package. Can just be the default scope. So are the 'public' methods in this class.